recipes/governor.rb

```diff
@@ -17,10 +19,8 @@
 # limitations under the License.
 #
 
-package 'cpufrequtils'
-
 (0..(node['cpu']['total'] - 1)).each do |i|
-  execute "set governator for CPU #{i}" do
+  execute "set governor for CPU #{i}" do
     command "cpufreq-set --cpu #{i} --governor #{node['cpu']['governor']}"
     action :run
     only_if "cpufreq-info --cpu #{i} --governors | grep #{node['cpu']['governor']}"
```

Comment:
Should this condition be inverted? I.e if governor already is set... then don't bother?

And would it make sense to guard against invalid arguments?

    only_if { %w(performance ondemand userspace conservative powersave).include? node['cpu']['governor'] }